lib/core/Canvas.js

```diff
@@ -427,11 +541,12 @@ Canvas.prototype.toggleMarker = function(element, marker) {
 };
 
 Canvas.prototype.getRootElement = function() {
-  if (!this._rootElement) {
-    this.setRootElement({ id: '__implicitroot', children: [] });
+  var plane = this.getActivePlane();
+  if (!plane.rootElement) {
+    this.setRootElement({ id: '__implicitroot' + plane.name, children: [] });
```

Comment:
This would currently cause issues both in the old and in the new properties panels as they check for an implicit root element through the ID. I never understood why we don't use a flag instead of using a special ID to mark implicit roots.

Reply:
Good catch, I was not aware of this. Let's fix it. I'd go ahead and add a flag and create PRs for the property panel as well